CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Contributing
+
+We have provided detailed documentation for ways in which you can
+contribute to Pulp here: https://docs.pulpproject.org/
```

Comment:
What about linking directly to our "Contributing" page:

https://docs.pulpproject.org/en/3.0/nightly/contributing/index.html

Also, I think we want to link to our 3.0 docs on master (vs our 2.y docs).

Reply:
I really wanted to do this, but I don't want this to constantly end up out of date and we currently have our stable current docs linking to the 2.y docs, and 3.0 is nightly only so I think we need to address that before I link to it here.

Reply:
I'm not sure what you mean by "constantly out of date". Can you explain? Regardless though I see less harm in linking to docs that might be a little out of date than linking to our pulp 2 docs.